frontend/include/chpl/resolution/scope-types.h

```diff
@@ -60,6 +61,55 @@ class IdAndFlags {
   /** A bit-set of the flags defined in the above enum */
```

Comment:
Let's improve this comment to say "representing the conjunction of the properties with set bits"